### PR TITLE
Page "About us" was added

### PR DESCRIPTION
**Pull request by Axedez**
[Trello Task](https://trello.com/c/mZuA2epY/45-about-us-page)

**CHECK LIST**
- [x] specs added
- [x] specs passed
- [x] code coverage >90%
- [ ] DB indexes added if needed
- [ ] seed.rb file is updated if needed
- [x] PR is reviewed manually again (to make sure you have 100% ready code)
- [x] PR doesn't have conflicts with main branch
- [ ] I've checked new feature as logged in and logged out user if needed
- [ ] "let!" is used in specs
- [x] screenshot of feature is added if task has UI
image
image
image

### CODE REVIEW

File: app/javascript/stylesheets/about.scss [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+.title {
+  color: grey;
+  margin-bottom: 0;
+}
+@media screen and (max-width: 650px) {
+	.column {
+		width: 100%;
+		display: block;
+	}
+}
```
Comment (mmarusyk) [inline app/javascript/stylesheets/about.scss:59]: Вирівняй блок

File: app/javascript/stylesheets/about.scss [hunk 2]
```diff
@@ -0,0 +1,59 @@
+@import url('https://fonts.googleapis.com/css2?family=David+Libre&display=swap');
+.about-title {
+  font-weight: bold;
+  margin: 30px;
+}
```
Comment (BogdanBusko) [inline app/javascript/stylesheets/about.scss:5]: enter

File: app/javascript/stylesheets/about.scss [hunk 3]
```diff
@@ -0,0 +1,59 @@
+@import url('https://fonts.googleapis.com/css2?family=David+Libre&display=swap');
```
Comment (BogdanBusko) [inline app/javascript/stylesheets/about.scss:1]: enter

File: app/javascript/stylesheets/about.scss [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+}
+.about-paragraph {
+  font-family: 'David Libre', serif;
+  text-align: center;
+  font-size: 2.3em;
+}
+.about-paragraph span{
+  font-weight: bold;
+}
+.row {
```
Comment (BogdanBusko) [inline app/javascript/stylesheets/about.scss:14]: alignment

File: app/javascript/stylesheets/about.scss [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+.about-title {
+  font-weight: bold;
+  margin: 30px;
+}
+.about-paragraph {
+  font-family: 'David Libre', serif;
+  text-align: center;
+  font-size: 2.3em;
+}
+.about-paragraph span{
```
Comment (BogdanBusko) [inline app/javascript/stylesheets/about.scss:11]: space after span

File: app/views/home/about.html.slim [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+        h2 #{t('about.vasyl_matsevko')}
+        p.title #{t('about.speciality')}
+        p matsevko.vasil@gmail.com   
+  .column
+    .card
+      img alt="Andrii Kozak Photo" src="/about_us_photos/andrii_koz.jpg"
+      .profile_container
+        h2 #{t('about.andrii_kozak')}
+        p.title #{t('about.speciality')}
+        p kozak-mail@gmail.com
```
Comment (mmarusyk) [inline app/views/home/about.html.slim:66]: Новий рядок в кінці

File: config/locales/ua.yml [hunk 7]
```diff
@@ -51,3 +51,17 @@ ua:
...
+    paragraph1: '"Цей сайт був розроблений креативними і цілеспрямованими учасниками'
+    paragraph2: 'академії."'
+    speciality: 'Розробник Ruby on rails'
+    andrii_lebukhorskyi: 'Андрій Лебухорський'
+    ivan_matsiyk: 'Іван Мацюк'
+    ihor_moskvyn: 'Ігор Москвин'
+    marian_rebeha: "Мар'ян Ребега"
+    ivan_danyliv: 'Іван Данилів'
+    mykola_ukhanskyi: 'Микола Уханський'
+    mykhailo_marusyk: 'Михайло Марисюк'
```
Comment (mmarusyk) [inline config/locales/ua.yml:65]:
```suggestion
    mykhailo_marusyk: 'Михайло Марусик'
```

File: app/views/shared/_footer.html.slim [hunk 8]
```diff
@@ -9,7 +9,7 @@ footer.footer.mt-2.pt-2.mb-0
         ul.m-0.p-0
           li
             | - 
-            a[href="#"] #{t('footer.link_about_us')}
+            a[href="/about"] #{t('footer.link_about_us')}
```
Comment (BogdanBusko) [inline app/views/shared/_footer.html.slim:12]: link_to

File: config/locales/en.yml [hunk 9]
```diff
@@ -43,11 +43,25 @@ en:
...
     all_text: 'The change you wanted was rejected'
 
   time:
     formats:
       long: "%d/%m/%Y %H:%M"
+  about:
+    header: 'About us'
+    paragraph1: '"This cite was developed by creative and self-motivated members of'
+    paragraph2: 'academy."'
+    speciality: 'Ruby on rails developer'
```
Comment (mmarusyk) [inline config/locales/en.yml:58]:
```suggestion
    speciality: 'Ruby on Rails developer'
```

File: app/views/shared/_header.html.slim [hunk 10]
```diff
@@ -1,12 +1,12 @@
 header.header.w-100
   nav.navbar.navbar-expand-lg.navbar-dark.navbar-header
-    a.nav-name.navbar-brand.text-black href="#" #{t('header.nav_name')}
+    a.nav-name.navbar-brand.text-black href="/#{I18n.locale}/" #{t('header.nav_name')}
```
Comment (BogdanBusko) [inline app/views/shared/_header.html.slim:3]: link_to

File: app/javascript/stylesheets/about.scss (hunk 1 above)
Comment (Axedez) [reply app/javascript/stylesheets/about.scss:59]: image
В редакторі все ок, це гіт розкидав

File: config/routes.rb [hunk 11]
```diff
@@ -3,10 +3,11 @@
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
   scope '(:locale)', locale: /#{I18n.available_locales.join("|")}/, defaults: { locale: I18n.default_locale } do
     root 'home#index', as: 'home'
+    get 'about' => 'home#about'
 
-    get "/404", to: "errors#not_found"
-    get "/422", to: "errors#unacceptable"
-    get "/500", to: "errors#server_error"
+    get "/404" => "errors#not_found"
```
Comment (BogdanBusko) [inline config/routes.rb:8]: live prev code

File: spec/controllers/home_controller_spec.rb [hunk 12]
```diff
@@ -1,8 +1,14 @@
...
+
   describe '#index' do
     subject { get :index }
 
     it { is_expected.to be_successful }
   end
+
+  describe 'render view #about' do
+    it { is_expected.to render_template(:about) }
+  end
```
Comment (mmarusyk) [inline spec/controllers/home_controller_spec.rb:13]:
```suggestion
  describe '#index' do
    subject { get :index }

    it { is_expected.to be_successful }
  end

  describe 'render view #about' do
    before { get :about }
    
    it { is_expected.to render_template(:about) }
  end
```

File: app/controllers/application_controller.rb [hunk 13]
```diff
@@ -7,6 +7,10 @@ def switch_locale(&action)
     I18n.with_locale(locale, &action)
   end
 
+  def default_url_options
+    { locale: I18n.locale }
```
Comment (vikdotdev) [inline app/controllers/application_controller.rb:11]: не треба цього, отаке є https://github.com/IF-115-Ruby/OSBB/pull/30/files#diff-21497849d8f00507c9c8dcaf6288b136R4

File: app/views/home/about.html.slim [hunk 14]
```diff
@@ -0,0 +1,66 @@
...
+  .column
+    .card
+      img alt="Ivan Danyliv Photo" src="/about_us_photos/ivan_dan.jpg"
+      .profile_container
+        h2 #{t('about.ivan_danyliv')}
+        p.title #{t('about.speciality')}
+        p danyliv04@gmail.com
+  .column
+    .card
+      img alt="Mykola Ukhanskyi Photo" src="/about_us_photos/mykola_ukh.jpg"
```
Comment (BogdanBusko) [inline app/views/home/about.html.slim:41]: image_tag

File: app/views/home/about.html.slim [hunk 15]
```diff
@@ -0,0 +1,66 @@
...
+        h2 #{t('about.ihor_moskvyn')}
+        p.title #{t('about.speciality')}
+        p ihor.moskvyn@gmail.com
+  .column
+    .card
+      img alt="Marian Rebeha Photo" src="/about_us_photos/marian_reb.jpg"
+      .profile_container
+        h2 #{t('about.marian_rebeha')}
+        p.title #{t('about.speciality')}
+        p marikrebega@gmail.com
```
Comment (BogdanBusko) [inline app/views/home/about.html.slim:31]: mail_to

File: app/javascript/stylesheets/about.scss (hunk 1 above)
Comment (vikdotdev) [reply app/javascript/stylesheets/about.scss:59]: то не гіт розкидав, то таби в тебе в редакторі, попробуй мишкою виділити і побачиш

File: app/javascript/stylesheets/about.scss [hunk 16]
```diff
@@ -0,0 +1,59 @@
...
+.card {
+	box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+}
+.card img {
+	width: 100%;
+}
+.profile_container {
+	padding: 0 16px;
+	&::after {
+		content: "";
```
Comment (pereslop) [inline app/javascript/stylesheets/about.scss:37]: too much white space here

File: app/views/home/about.html.slim [hunk 17]
```diff
@@ -0,0 +1,66 @@
+h1.about-title #{t('about.header')}
+p.about-paragraph #{t('about.paragraph1')} 'Ruby on Rails' <span>SoftServe</span>#{t('about.paragraph2')}
+.row
+  .column
+    .card
+      img alt="Andrii Lebukhorskyi Photo" src="/about_us_photos/andrii_leb.jpg"
```
Comment (pereslop) [inline app/views/home/about.html.slim:6]: look like you have here too much identical blocks, DRY it with view partials

File: config/routes.rb (hunk 11 above)
Comment (Axedez) [reply config/routes.rb:8]: Ок, тільки скажи будь-ласка в чому причина, щоб знав на майбутнє

File: app/views/home/about.html.slim [hunk 18]
```diff
@@ -0,0 +1,12 @@
...
+.row
+  = render partial: 'shared/about_profile', :locals => {name: 'andrii_lebukhorskyi', mail: 'andrey.lebuhorskyy@gmail.com'}
+  = render partial: 'shared/about_profile', :locals => {name: 'ivan_matsiyk', mail: 'ivan.matsiyk18092002@gmail.com'} 
+  = render partial: 'shared/about_profile', :locals => {name: 'ihor_moskvyn', mail: 'ihor.moskvyn@gmail.com'}
+  = render partial: 'shared/about_profile', :locals => {name: 'marian_rebeha', mail: 'marikrebega@gmail.com'}
+  = render partial: 'shared/about_profile', :locals => {name: 'ivan_danyliv', mail: 'danyliv04@gmail.com'}
+  = render partial: 'shared/about_profile', :locals => {name: 'mykola_ukhanskyi', mail: 'ukhanskyi@gmail.com'}
+  = render partial: 'shared/about_profile', :locals => {name: 'mykhailo_marusyk', mail: 'misha.marusyk@gmail.com'}
+  = render partial: 'shared/about_profile', :locals => {name: 'vasyl_matsevko', mail: 'matsevko.vasil@gmail.com'}
+  = render partial: 'shared/about_profile', :locals => {name: 'andrii_kozak', mail: 'kozak-mail@gmail.com'}
```
Comment (Ukhanskyi) [inline app/views/home/about.html.slim:12]: here need new line

File: app/views/shared/_about_profile.html.slim [hunk 19]
```diff
@@ -0,0 +1,7 @@
+.column
+  .card
+    = image_tag  "/about_us_photos/#{name}.jpg", alt: "#{t("about.#{name}")} #{t('about.photo')}"
+    .profile_container
+      h2 #{t("about.#{name}")}
+      p.title #{t('about.speciality')}
+      = mail_to mail, mail
```
Comment (Ukhanskyi) [inline app/views/shared/_about_profile.html.slim:7]: and here

File: app/views/shared/_header.html.slim [hunk 20]
```diff
@@ -1,12 +1,12 @@
 header.header.w-100
   nav.navbar.navbar-expand-lg.navbar-dark.navbar-header
-    a.nav-name.navbar-brand.text-black href="#" #{t('header.nav_name')}
+    = link_to "#{t('header.nav_name')}", home_path, class: "nav-name navbar-brand text-black"
```
Comment (Ukhanskyi) [inline app/views/shared/_header.html.slim:3]: probably write like here?
= link_to home_path, class: 'nav-name navbar-brand text-black'
  = t('header.nav_name')

File: app/views/home/about.html.slim (hunk 18 above)
Comment (vasya539) [inline app/views/home/about.html.slim:12]: look you have pretty much of similar code, try to refactor it

**Review by Swooty18 — APPROVED** [review]

File: app/views/shared/_about_profile.html.slim [hunk 21]
```diff
@@ -0,0 +1,7 @@
+.column
+  .card
+    = image_tag  "/about_us_photos/#{name}.jpg", alt: "#{t("about.#{name}")} #{t('about.photo')}"
+    .profile_container
+      h2 #{t("about.#{name}")}
```
Comment (pereslop) [inline app/views/shared/_about_profile.html.slim:5]: why do you use interpolation here? can just write `h2 = t("about.#{name}")`

File: app/views/shared/_footer.html.slim [hunk 22]
```diff
@@ -8,8 +8,8 @@ footer.footer.mt-2.pt-2.mb-0
         h5.mt-lg-0.mt-sm-3 #{t('footer.links')}
         ul.m-0.p-0
           li
-            | - 
-            a[href="#"] #{t('footer.link_about_us')}
+            | -
+            a = link_to "#{t('footer.link_about_us')}", about_path
```
Comment (pereslop) [inline app/views/shared/_footer.html.slim:12]: interpolation
`a = link_to t('footer.link_about_us'), about_path`
or just
`a = link_to t('footer.link_about_us'), about_path`
#link_to method aready should create link

**Review by matsevich — APPROVED** [review]

File: app/javascript/stylesheets/about.scss [hunk 23]
```diff
@@ -0,0 +1,71 @@
...
+  text-align: center;
+  font-size: 2.3em;
+}
+
+.about-paragraph span {
+  font-weight: bold;
+}
+
+.row {
+	display: flex;
```
Comment (pereslop) [inline app/javascript/stylesheets/about.scss:19]: Extra tab

File: app/models/user.rb [hunk 24]
```diff
@@ -13,6 +13,41 @@ class User < ApplicationRecord
 
   mount_uploader :avatar, AvatarUploader
 
+  TEAM_MEMBERS = [{ name: 'andrii_lebukhorskyi',
+                    email: 'andrey.lebuhorskyy@gmail.com' },
```
Comment (chu4) [inline app/models/user.rb:17]: wrong place for constant think of a different place

Comment (chu4) [reply app/models/user.rb:17]: its has nothing to do with user model

File: app/javascript/stylesheets/about.scss (hunk 23 above)
Comment (Axedez) [reply app/javascript/stylesheets/about.scss:19]: I dont understand whats wrong with my about.scss. In visual studio all tabs in correct form

**Review by marikrebega — CHANGES_REQUESTED** [review]
Maybe move developers photo to .gitignore?

**Review by marikrebega — APPROVED** [review]

**Review by mmarusyk — APPROVED** [review]

**Review by chu4 — APPROVED** [review]

**Pull request merged**